app/js/lib/foxbox/settings.js

```diff
@@ -29,29 +29,26 @@ const storage = localStorage ? localStorage : {
   clear: () => {},
 };
 
-export default class Settings extends Model {
+export default class Settings extends EventDispatcher {
   constructor() {
+    super(['session-changed', 'polling-setting-changed']);
```

Comment:
Settings essentially started as a wrapper around local storage. I still think that it's a good thing to have all local storage elements managed in the same class. But I understand your point of view.